src/items.zig

```diff
@@ -982,7 +984,7 @@ pub fn register(_: []const u8, texturePath: []const u8, replacementTexturePath:
 	defer itemListSize += 1;
 
 	newItem.init(arena.allocator(), texturePath, replacementTexturePath, id, zon);
-	reverseIndices.put(newItem.id, .{.index = itemListSize}) catch unreachable;
+	reverseIndices.put(newItem.id, @enumFromInt(itemListSize)) catch unreachable;
```

Comment:
Idea for a future refactor: How about putting these global variables into the enum and adding a `nextIndex` function?

Reply:
I find presence of such function in interface of most `*Index` structs a bit awkward - realistically only context where it is walid to call this function is during asset initialization. I think it is appropriate to do that for `ModelIndex`, but for other structs I am not so sure. As for the global variable it should be fine to move it, but I need to have a look at individual cases.